export/data/books.json

```diff
@@ -0,0 +1,80 @@
+[
```

Comment:
Why is this `export` directory committed? I thinks that was for testing?

Reply:
No, this folder contains the exported data that must be downloaded.